gdnative-derive/src/methods.rs

```diff
@@ -12,6 +12,7 @@ pub(crate) struct ClassMethodExport {
 pub(crate) struct ExportMethod {
     pub(crate) sig: Signature,
     pub(crate) args: ExportArgs,
+    pub(crate) rpc: String,
```

Comment:
There is already a `ExportArgs` type for parsed attribute arguments. Please move the `rpc` field there and use a Rust enum for the parsed value, so that no further parsing will be necessary down the line.